README.md

```diff
@@ -65,6 +65,8 @@ This means we follow the same installation than above, but using the
 1.2.0
 ```
 
+> if you have problem with installing odoc by `opam install odoc` the try to use `opam pin add odoc.dev git+https://github.com/ocaml/odoc`
```

Comment:
Even if people do install current stable odoc successfully with opam, Reason syntax will not be supported! I think this should say something like: "BuckleScript support currently requires the latest development version of odoc." and replace the `opam install odoc` line with the pin command instead.

Reply:
👍